packages/zcli-core/src/lib/auth.ts

```diff
@@ -28,23 +29,26 @@ export default class Auth {
     if (ZENDESK_OAUTH_TOKEN) {
       return `Bearer ${ZENDESK_OAUTH_TOKEN}`
     } else if (ZENDESK_EMAIL && ZENDESK_API_TOKEN) {
-      return this.createBasicAuthToken(`${ZENDESK_EMAIL}/token`, ZENDESK_API_TOKEN)
+      return this.createBasicAuthToken(`${ZENDESK_EMAIL}`, ZENDESK_API_TOKEN)
     } else if (ZENDESK_EMAIL && ZENDESK_PASSWORD) {
-      return this.createBasicAuthToken(ZENDESK_EMAIL, ZENDESK_PASSWORD)
+      return this.createBasicAuthToken(ZENDESK_EMAIL, ZENDESK_PASSWORD, SecretType.PASSWORD)
```

Comment:
This check is kept for those users that still only have `ZENDESK_PASSWORD` in their systems (e.g. cicd, local, etc) to properly fire an error that it is not supported.